### PR TITLE
Stack queue pseudo

### PR DESCRIPTION
**Pull request by snur206**
[x] Top-level README “Table of Contents” is updated
 - [x] README for this challenge is complete
       - [x] Summary, Description, Approach & Efficiency, Solution
       - [x] Picture of whiteboard
       - [x] Link to code
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
       - [ ] “Happy Path” - Expected outcome
       - [ ] Expected failure
       - [x] Edge Case (if applicable/obvious)

### CODE REVIEW

File: java/datastructures/lib/src/main/java/codechallenges/linkedlist/pseudoQueue.java [hunk 1]
```diff
@@ -11,8 +11,21 @@ public void enqueue(int val) {
   }
 
   public void dequeue() {
-    while(stack2.isEmpty()) {
-
+    if(stack2.isEmpty()) {
+      if(stack1.isEmpty());
+    } else {
```
Comment (reyjroliva) [inline java/datastructures/lib/src/main/java/codechallenges/linkedlist/pseudoQueue.java:16]: can remove if statements and move right into the while loop

File: java/datastructures/lib/src/main/java/codechallenges/linkedlist/pseudoQueue.java [hunk 2]
```diff
@@ -11,8 +11,21 @@ public void enqueue(int val) {
...
+      while(!stack1.isEmpty()){
+        stack2.push(stack1.pop());
+      }
```
Comment (reyjroliva) [inline java/datastructures/lib/src/main/java/codechallenges/linkedlist/pseudoQueue.java:19]: You've successfully moved everything from stack1 (hold) into stack2 (remove). Just need to complete the logic. 

Lines 17-19
- Push all data to RemoveStack from the HoldStack by popping the values off the HoldStack
 
After (outside) the while loop:

- Pop top of RemoveStack and place in a temp variable to return later
`Node temp = stack2.pop();`

- Push all data back into the HoldStack by popping values off the RemoveStack
- Keeps the integrity of our DataStructure and makes it ready to have new values added
`while(!stack2.isEmpty()) {`
`stack1.push(stack2.pop());`
`}`

- Return value held in temp Node (was popped off the RemoveStack)
`return temp;`
